infrastructure/modules/app_config/main.tf

```diff
@@ -1,7 +1,9 @@
 locals {
-  is_sandbox_or_test = contains(["ndra", "ndrb", "ndrc", "ndrd", "ndr-test"], terraform.workspace)
-  current_config_path = (local.is_sandbox_or_test ? "${path.module}/configurations/2024-02-21-dev.json" :
-  "${path.module}/configurations/2024-02-21.json")
+  is_sandbox_dev_or_test = contains(["ndra", "ndrb", "ndrc", "ndrd", "ndr-dev", "ndr-test"], terraform.workspace)
+  is_preprod             = contains(["pre-prod"], terraform.workspace)
+  current_config_path = (local.is_sandbox_dev_or_test ? "${path.module}/configurations/2024-02-21-dev.json"
+    : local.is_preprod ? "${path.module}/configurations/2024-05-21-preprod.json"
+  : "${path.module}/configurations/2024-02-21-prod.json")
```

Comment:
Is there any way we could clean this up slightly? Instead of having the `is_preprod` flag, we could say if not `is_sandbox_dev_or_test` instead, then our config path could look like

```
current_config_path =  local.is_sandbox_dev_or_test ? "${path.module}/configurations/2024-02-21-dev.json"
    : "${path.module}/configurations/2024-05-21-${terraform.workspace}.json"
```